pkg/util/encryption.go

```diff
@@ -17,7 +17,7 @@ const saltLength = 8
 // Decrypt decrypts a payload with a given secret.
 // Deprecated. Do not use it.
 // Use encryption.Service instead.
-var Decrypt = func(payload []byte, secret string) ([]byte, error) {
+func Decrypt(payload []byte, secret string) ([]byte, error) {
```

Comment:
Remind me, why are we keeping these around if we're not supposed to use them?

Reply:
We're still using them in cli and in [alerting migrations](https://github.com/grafana/grafana/blob/0e87f3e8176a1b5f100c8e12718517bba69dfe7a/pkg/services/sqlstore/migrations/ualert/securejsondata.go) Please, take a look at this comment https://github.com/grafana/grafana/pull/38865#discussion_r716637928 for more details.

Reply:
Right, thanks, I remember you telling me about that now, thanks.